src/components/DropMenu/template/UserMenu.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import cn from 'classnames';
+import { useAppDispatch } from '../../../app/hooks';
+import { logOutUserAction } from '../../../app/modules/auth/actions';
+import { ReactComponent as SettingsSvg } from '../../../assets/settings.svg';
+import { ReactComponent as ExitSvg } from '../../../assets/exit.svg';
+import { ReactComponent as SunSvg } from '../../../assets/sun.svg';
+import { ReactComponent as MoonSvg } from '../../../assets/moon.svg';
+
+const STYLE_MENU_ITEM =
+  'flex fill-[#C6C5C5] items-center px-4 py-2 rounded-full cursor-pointer duration-100 hover:bg-gray-2 hover:text-white hover:fill-[#D9D9D9] select-none';
+
+const UserMenu = () => {
+  const [theme, setTheme] = useState<string>('white');
+
+  const dispatch = useAppDispatch();
+
+  const logOut = () => {
+    dispatch(logOutUserAction());
+  };
+
+  const USER_NAME = localStorage.getItem('userName');
```

Comment:
Use camelCase inside the component. SCREAMING_SNAKE_CASE using only outside the component